### ISSUE TITLE
Neo4j Output NPE when no rows

### ISSUE THREAD

**Issue by rhaces** [knowbi/knowbi-pentaho-pdi-neo4j-output#88]
When Neo4j Output gets no rows, it throws an NPE:

```2019/01/29 21:36:08 - load_activityLevel.0 - ERROR (version 8.2.0.0-342, build 8.2.0.0-342 from 2018-11-14 10.30.55 by buildguy) : Unexpected error
2019/01/29 21:36:08 - load_activityLevel.0 - ERROR (version 8.2.0.0-342, build 8.2.0.0-342 from 2018-11-14 10.30.55 by buildguy) : java.lang.NullPointerException
2019/01/29 21:36:08 - load_activityLevel.0 - 	at bi.know.kettle.neo4j.steps.cypher.Cypher.processRow(Cypher.java:149)
2019/01/29 21:36:08 - load_activityLevel.0 - 	at org.pentaho.di.trans.step.RunThread.run(RunThread.java:62)
2019/01/29 21:36:08 - load_activityLevel.0 - 	at java.lang.Thread.run(Thread.java:748)
```

**Comment by mattcasters** [comment]
Thanks for the report Rodrigo, really appreciate it!

**Comment by mattcasters** [comment]
A fix was released in version 4.0.0.
